pom.xml

```diff
@@ -136,6 +136,7 @@
 		<tidy-maven-plugin.version>1.1.0</tidy-maven-plugin.version>
 		<license-maven-plugin.version>4.1</license-maven-plugin.version>
 		<code-assert.version>0.9.15</code-assert.version>
+        <ipaddress.version>5.3.3</ipaddress.version>
```

Comment:
formatting

Reply:
This one's interesting as it's a "tab" vs. "space" thing.  Seems we have default tab indents for XML files in the project, which I should probably switch to spaces to prevent things like this.